dask/dataframe/io/sql.py

```diff
@@ -211,3 +212,16 @@ def _read_sql_chunk(q, uri, meta, engine_kwargs=None, **kwargs):
         return meta
     else:
         return df.astype(meta.dtypes.to_dict(), copy=False)
+
+
+def to_sql(df, *args, compute=True, **kwargs):
+    df._meta.to_sql(*args, **kwargs)
+
+    # Partitions should always append to the empty table created from `meta` above
+    kwargs["if_exists"] = "append"
```

Comment:
For the future...
The SQL server may not support multiple partitions trying to append at the same time, and certainly the order of the rows cannot be guaranteed. It may be worthwhile to allow for a lock or chain the partition dependencies (make each partition write depend on the previous one) for a more constrained version of this function.